src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 2;
+  const LONG_TERM = 7;
+  const PRICE = 40;
+  let sale = 0;
+
+  if (days > SHORT_TERM && days < LONG_TERM) {
+    sale = 20;
+  }
```

Comment:
The condition `days > SHORT_TERM && days < LONG_TERM` does not correctly implement the discount logic. According to the task description, the discount should be applied if the car is rented for 3 or more days. The condition should be `days >= SHORT_TERM`.